sw/device/tock/dif/src/lib.rs

```diff
@@ -0,0 +1,14 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+//! Bindings for OpenTitan DIF libaries
+
+#![no_std]
+#![crate_name = "dif"]
+#![crate_type = "rlib"]
+
+#[allow(nonstandard_style)]
+pub mod dif_bind;
```

Comment:
Consider instead:
```
pub mod dif_bind {
  include!(...);
}
```

Reply:
+1 to this and the comment above.

Reply:
Sounds good, thanks.

Reply:
Honestly I find it really confusing when in the [documentation](https://doc.rust-lang.org/std/macro.include.html) for `std`:
`Macro std::include`
you click on the [src](https://doc.rust-lang.org/src/core/macros/mod.rs.html#1223-1226), and it actually turns out to be `core`.

Reply:
Done.